cloudos_cli/__main__.py

```diff
@@ -1028,21 +1033,21 @@ def list_projects(apikey,
         print('\t' + str(cl) + '\n')
         print('\tSearching for projects in the following workspace: ' +
               f'{workspace_id}')
-    my_projects_r = cl.get_project_list(workspace_id, verify_ssl)
+    # Check if the user provided the --page option
+    ctx = click.get_current_context()
+    if ctx.get_parameter_source('page') == click.core.ParameterSource.DEFAULT:
+        get_all = True
+    else:
+        get_all = False
+    my_projects_r = cl.get_project_list(workspace_id, verify_ssl, page=page, get_all=get_all)
     if output_format == 'csv':
```

Comment:
```suggestion
    my_projects_r = cl.get_project_list(workspace_id, verify_ssl, page=page, get_all=get_all)
    if len(my_projects_r) == 0:
        if ctx.get_parameter_source('page') == click.core.ParameterSource.DEFAULT:
            print('\t[Message] A total of 0 projects collected. This is likely because your workspace has no projects created yet.')
        else:
            print('\t[Message] A total of 0 projects collected. This is likely because the --page you requested does not exist. Please, try a smaller number for --page or collect all the projects by not using --page parameter.')
    elif output_format == 'csv':
```
Could we also add something similar in cloudos job list? Thanks!